scripts/runs/webshop_budget_jobs.sh

```
@@ -0,0 +1,57 @@
+    trainer.save_freq=50 \
+    agent_proxy.max_turn="${turns}" \
+    actor_rollout_ref.rollout.max_model_len="${max_len}" actor_rollout_ref.rollout.max_num_batched_tokens="${max_tok}" \
+    trainer.default_local_dir="${out_dir}" \
+    trainer.max_actor_ckpt_to_keep=4 \
+    trainer.max_critic_ckpt_to_keep=4 \
+    custom_envs.WebShop.max_actions_per_traj="${turns}" \
+    actor_rollout_ref.actor.use_ref=False \
+    trainer.nnodes=1
+}
+
+main() {
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this function.

(SC1009)